src/crs/projected/eqareaalbers.jl

```diff
@@ -0,0 +1,155 @@
+  Albers{latₒ,lat₁,lat₂,Datum,Shift()}(args...)
+
+Albers(args...) = Albers{NAD83}(args...)
+
+Base.convert(
+  ::Type{Albers{latₒ,lat₁,lat₂,Datum,Shift,M}},
+  coords::Albers{latₒ,lat₁,lat₂,Datum,Shift}
+) where {latₒ,lat₁,lat₂,Datum,Shift,M} = Albers{latₒ,lat₁,lat₂,Datum,Shift,M}(coords.x, coords.y)
+
+constructor(::Type{<:Albers{latₒ,lat₁,lat₂,Datum,Shift}}) where {latₒ,lat₁,lat₂,Datum,Shift} =
+  Albers{latₒ,lat₁,lat₂,Datum,Shift}
+
+lentype(::Type{<:Albers{latₒ,lat₁,lat₂,Datum,Shift,M}}) where {latₒ,lat₁,lat₂,Datum,Shift,M} = M
+
+==(
+  coords₁::Albers{latₒ,lat₁,lat₂,Datum,Shift},
+  coords₂::Albers{latₒ,lat₁,lat₂,Datum,Shift}
+) where {latₒ,lat₁,lat₂,Datum,Shift} = coords₁.x == coords₂.x && coords₁.y == coords₂.y
+
+# ------------
+# CONVERSIONS
+# ------------
+
+# Adapted from PROJ coordinate transformation software
+# Initial PROJ 4.3 public domain code was put as Frank Warmerdam as copyright
+# holder, but he didn't mean to imply he did the work. Essentially all work was
+# done by Gerald Evenden.
+# Authors of the original algorithm: Gerald Evenden and Thomas Knudsen
+# reference code: https://github.com/OSGeo/PROJ/blob/master/src/projections/aea.cpp
+
+inbounds(::Type{<:Albers}, λ, ϕ) =
+  -π ≤ λ ≤ π && deg2rad(90) ≤ ϕ ≤ deg2rad(90)
+
+function formulas(::Type{<:Albers{latₒ,lat₁,lat₂,Datum}}, ::Type{T}) where {latₒ,lat₁,lat₂,Datum,T}
+  # Constants
+  🌎 = ellipsoid(Datum)
+  e = T(eccentricity(🌎))
+  a = numconvert(T, majoraxis(🌎))
+
+  # Latitude origin
+  ϕₒ = T(ustrip(deg2rad(latₒ)))
+
+  # Standard parallels
+  ϕ₁ = T(ustrip(deg2rad(lat₁)))
+  ϕ₂ = T(ustrip(deg2rad(lat₂)))
+
+  m₁ = hm(ϕ₁, e)
+  m₂ = hm(ϕ₂, e)
+  α₁ = hα(ϕ₁, e)
+  α₂ = hα(ϕ₂, e)
+  αₒ = hα(ϕₒ, e)
+  n = (m₁^2 - m₂^2) / (α₂ - α₁)
+  C = m₁^2 + n * α₁
+
+  Θ = n * λ
+  ρ = a * sqrt(C - n * hα(ϕ, e)) / n
+  if ρ < 0
+    throw(ArgumentError("coordinates outside of the projection domain"))
+  end
+  ρₒ = sqrt(a * (C - n * αₒ)) / n
+  function fx(λ, ϕ)
+    ρₒ - ρ * cos(Θ)
+  end
+
+  function fy(λ, ϕ)
+    ρ * sin(Θ)
+  end
+
+  fx, fy
+end
+
+# backward projection formulas
+
+function backward(::Type{<:Albers{latₒ,lat₁,lat₂,Datum}}, x, y) where {latₒ,lat₁,lat₂,Datum}
+  🌎 = ellipsoid(Datum)
+  e = oftype(x, eccentricity(🌎))
+  e² = oftype(x, eccentricity²(🌎))
+  ϕₒ = oftype(x, ustrip(deg2rad(latₒ)))
+  ϕ₁ = oftype(x, ustrip(deg2rad(lat₁)))
+  ϕ₂ = oftype(x, ustrip(deg2rad(lat₂)))
+  αₒ = hα(ϕₒ, e)
+  ρₒ = sqrt(a * (C - n * αₒ)) / n
+  α₁ = hα(ϕ₁, e)
+  α₂ = hα(ϕ₂, e)
+
+  m₁ = hm(ϕ₁, e)
+  m₂ = hm(ϕ₂, e)
+  n = (m₁^2 - m₂^2) / (α₂ - α₁)
+  C = m₁^2 + n * α₁
+
+  θ = atan2(x, ρₒ - y)
+  ρ = sqrt(x^2 + (ρₒ - y)^2)
+  α′ = (C - (ρ^2 * n^2) / a^2) / n
+  β′ = asin(α′ / (1 - (1 - e) / (2 * e) * log((1 - e) / (1 + e))))
+
+  λ = θ / n
+  ϕ =
+    β′ +
+    (e^2 / 3 + 31 * e^4 / 180 + 517 * e^6 / 5040) * sin(2 * β′) +
+    (23 * e^4 / 360 + 251 * e^6 / 3780) * sin(4 * β′) +
+    (761 * e^6 / 45360) * sin(6 * β′)
+
+  λ, ϕ
```

Comment:
```suggestion
  🌎 = ellipsoid(Datum)
  e = oftype(x, eccentricity(🌎))
  e² = oftype(x, eccentricity²(🌎))
  ϕₒ = oftype(x, ustrip(deg2rad(latₒ)))
  ϕ₁ = oftype(x, ustrip(deg2rad(lat₁)))
  ϕ₂ = oftype(x, ustrip(deg2rad(lat₂)))

  ρₒ = sqrt(C - n * αₒ) / n
  m₁ = hm(ϕ₁, e)
  m₂ = hm(ϕ₂, e)
  αₒ = hα(ϕₒ, e)
  α₁ = hα(ϕ₁, e)
  α₂ = hα(ϕ₂, e)
  n = (m₁^2 - m₂^2) / (α₂ - α₁)
  C = m₁^2 + n * α₁

  θ = atan2(x, ρₒ - y)
  ρ = sqrt(x^2 + (ρₒ - y)^2)
  α′ = (C - (ρ^2 * n^2)) / n
  β′ = asin(α′ / (1 - (1 - e) / (2 * e) * log((1 - e) / (1 + e))))

  λ = θ / n
  ϕ =
    β′ +
    (e^2 / 3 + 31 * e^4 / 180 + 517 * e^6 / 5040) * sin(2 * β′) +
    (23 * e^4 / 360 + 251 * e^6 / 3780) * sin(4 * β′) +
    (761 * e^6 / 45360) * sin(6 * β′)

  λ, ϕ
```
Division by `a` is also done in a preprocessing step.
Adjust the order in which constants are defined.